package.json

```diff
@@ -19,6 +19,11 @@
 		"index.js",
 		"index.d.ts"
 	],
+	"exports": {
+		".": "./index.js",
+		"./filenamify": "./filenamify.js",
+		"./filenamify-path": "./filenamify-path.js"
```

Comment:
This is not what we discussed in the issue.

Reply:
I'm not 100% sure what you mean here, but if you meant that you don't want to provide the `filenamify-path` export then I've just pushed a commit that removes it.

I've also changed the name of the `filenamify` export to `browser`, as suggested by you in the issue. Again, it's always easy to change it.

And if there's anything else you want changed, please feel free to tell me and I'll change it.

Reply:
Yeah, that's what I meant. Sorry if that was not clear.